pkg/syncer/testing/testing.go

```diff
@@ -225,14 +225,6 @@ func stripObject(obj runtime.Object) runtime.Object {
 		accessor.SetAnnotations(nil)
 	}
 
-	l := accessor.GetLabels()
-	if l != nil {
-		delete(l, "vcluster.loft.sh/managed-by")
```

Comment:
All the tests have been updated to check for `vcluster.loft.sh/managed-by` label.